Gruntfile.js

```diff
@@ -22,10 +21,14 @@ module.exports = function(grunt){
   require('load-grunt-tasks')(grunt);
 
   grunt.registerTask('default', ['test-local', 'build']);
-  grunt.registerTask('sauce', ['karma:sauce-linux', 'karma:sauce-osx', 'karma:sauce-windows']);
-  grunt.registerTask('test-local', ['eslint', 'mochaTest:all', 'karma:local']);
-  grunt.registerTask('test-local-silent', ['eslint', 'mochaTest:silent', 'karma:local']);
-  grunt.registerTask('test', ['eslint', 'mochaTest:all']);
+  grunt.registerTask('sauce', [
+    'karma:sauce-linux',
+    'karma:sauce-osx',
+    'karma:sauce-windows'
```

Comment:
That was already like that, so didn't want to modify any of our rules with this PR. 
But totally, we could definitely discuss all of those one-by-one so please feel free to lead the effort with issues/pr in this direction so that we can discuss each of those improvements. Thanks for that!